src/App.tsx

```diff
@@ -1,61 +1,124 @@
 import './App.scss';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import { Post } from './types/post';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+function getUserById(userId: number) {
+  return usersFromServer.find(user => user.id === userId) || null;
+}
+
+const todos = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUserById(todo.userId),
+}));
```

Comment:
[CODE KNOWLEDGE] - You are using a non-mutating array method `map`, so you don't need to create a copy of the array. Also, consider moving this data preparation logic to a separate function or even a separate file. It would make the code cleaner and easier to read.